src/workerd/io/worker-interface.capnp

```diff
@@ -147,14 +147,31 @@ struct QueueMessage @0x944adb18c0352295 {
   contentType @3 :Text;
 }
 
+struct QueueRetryBatch {
+  retry @0 :Bool;
+  union {
```

Comment:
My initial preference was to provide a bogus value or simply use a boolean. However, other team members were more inclined to use a union, especially since there were already some examples in the codebase utilizing unions with void fields. The decision to use the union appeared to be cleaner (the field does not show up in the resulting object) and less error-prone (queues code does not need to be aware of any special meanings associated with the `delaySeconds` field) and the overhead does not seem to be that big.